packages/dev/query/src/cli.js

```diff
@@ -366,6 +367,22 @@ function traverseBundle(v: string) {
   });
 }
 
+function listUnusedSymbols() {
+  for (let n of assetGraph.nodes.values()) {
+    if (n.type === 'asset') {
+      if (n.usedSymbols.size === 0) continue;
+      invariant(n.value.symbols != null);
+      let allSymbols = new Set([...n.value.symbols?.keys()]);
+      let unusedSymbols = setDifference(n.usedSymbols, allSymbols);
+      if (unusedSymbols.size === 0) continue;
+      console.log(
+        fromProjectPathRelative(n.value.filePath) + ' Does not use symbols: ',
```

Comment:
This is what you're computing here. (Not sure if this is from prototyping or if there was something unclear about the meaning of `unusedSymbols`)
```suggestion
        fromProjectPathRelative(n.value.filePath) + ' contains these unused exports: ',
```